### DISCUSSION TITLE
How to update circular list view

### DISCUSSION THREAD

**Discussion by nuununuun** [LanKuDot/game_modules#20] [Q&A] [answered]
Hi.
I am using your asset, and I found it is really awesome, thanks for your effort.
btw, I have but one question.

**What I want is update list's view on runtime, when list's elements are added or deleted.**
Is there any way to achieve this?

Thanks.

**Comment by LanKuDot** [comment, marked as answer]
Hi, the first idea came up to me was that made `ListBox.UpdateDisplayContent()` public. But the boxes holding the contents near the end will display incorrectly.
For example, There have 10 items and the current displaying contents are 8, 9, 0, 1, 2. If I add an new item, by invoking `UpdateDisplayContent()` will still display 8, 9, 0, 1, 2. But it should be 9, 10, 0, 1, 2.

**Reply by nuununuun** [reply]
Thanks!!
